source/backend/api/Models/Concepts/AcquisitionFile/AcquisitionFileChecklistItemMap.cs

```diff
@@ -8,7 +8,6 @@ public class AcquisitionFileChecklistItemMap : IRegister
         public void Register(TypeAdapterConfig config)
         {
             config.NewConfig<Entity.PimsAcquisitionChecklistItem, AcquisitionFileChecklistItemModel>()
-                .PreserveReference(true)
```

Comment:
Doing this was strictly optional but I like having things concise where possible.